packages/engine/tests/integration.rs

```diff
@@ -1,6 +1,11 @@
 //! The hEngine integration test suite runs a variety of specially-designed simulations and
 //! experiments of specific functionalities to verify outputs.
 
+/// Helper for parsing an experiment and run it.
+mod experiment;
+
+mod examples;
+mod popular;
```

Comment:
Not entirely sure about the naming of this module or PR. Do we think calling it popular is the right way of phrasing this? Especially since they're still simulations under the @hash user. 

I don't mind if we stick with it, but can we at least add some documentation explaining the choice and what we're actually testing here, (at a glance a new user should be able to understand what the difference is between examples, popular, and units)

Reply:
I agree. As these are still examples but at other places we could still add them under `examples`.